### DISCUSSION TITLE
editor repeat any key i perss/click

### DISCUSSION THREAD

**Discussion by hafsabajwa** [codesandbox/codesandbox-client#8300] [Q&A]
whatever key I
![a](https://github.com/codesandbox/codesandbox-client/assets/65125439/30863fb5-6724-42a2-9ecd-20f27000595f)
![bfs](https://github.com/codesandbox/codesandbox-client/assets/65125439/905f809c-7fe8-413c-94f1-ee1b33be16a5)
 press or click anywhere on the sandbox editor it just repeats everything several times. i have attached the screnshot as well.

**Comment by JamesACS** [comment]
Hello there, 

There is nothing on our side that should cause this - have you encountered this behaviour on other sites, or are you using a wireless keyboard?

**Comment by hafsabajwa** [comment]
no, i am not using a wireless key board. The problem is on chrome and
firefox browsers. on Edge, it works fine.

**Comment by JamesACS** [comment]
That's interesting @hafsabajwa - as Chrome and Edge run on Chromium, where Firefox doesn't which would imply this is a device/operating system issue. 

As a troubleshooting step, please could you clear your cookies and cache on the effected browsers?

**Comment by hafsabajwa** [comment]
Thanks, I will!
